backend/asm_targets/asm_directives.ml

```diff
@@ -152,8 +144,10 @@ module Make (A : Asm_directives_intf.Arg) : Asm_directives_intf.S = struct
     (* Stop dsymutil complaining about empty __debug_line sections (produces
        bogus error "line table parameters mismatch") by making sure such
        sections are never empty. *)
-    let file_num = A.get_file_num "none" in
-    loc ~file_num ~line:1 ~col:1;
+    (* The following line is commented out because it adds a loc directive in a
```

Comment:
You should also include the comment `Stop dsymutil [..] empty.` in the commented out code, otherwise it's just confusing